api/v1/openapi.yaml

```diff
@@ -2613,6 +2626,9 @@ definitions:
           Update all services selecting the backends with their given states
           (id and frontend are ignored)
         type: boolean
+      weightsUpdated:
```

Comment:
Is this required to be defined in the API yaml, or can it be derived from backend-weights?

Reply:
removed